src/creature_control.h

```diff
@@ -410,6 +407,11 @@ unsigned short shot_shift_z;
     unsigned char stopped_for_hand_turns;
     long following_leader_since;
     unsigned char follow_leader_fails;
+    unsigned long timebomb_countdown;
+    unsigned char timebomb_radius; //todo cleanup
```

Comment:
we do still need to clean this up if we do not use this but get values from the config.